docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -18,7 +18,7 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
 
 ### Deprecation
 
--
+* `util.MachinesByCreationTimestamp` has been deprecated, as we are now using `util/collections.machinesByCreationTimestamp`
```

Comment:
```suggestion
* `util.MachinesByCreationTimestamp` has been deprecated and will be removed in a future release.
```
nit. I would write it like this. I think from an API consumer perspective it doesn't matter that we are still using a copy internally.

Reply:
Done.